### PR TITLE
Use golang1.14.4 to compile AddonResizer

### PR DESCRIPTION
**Pull request by jkaniuk**
More aggressive GC should help with memory usage

/cc wojtek-t bskiba

### CODE REVIEW

File: addon-resizer/Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 ARCH ?= amd64
 
 GOARM=7
-GOLANG_VERSION = 1.12.1
+GOLANG_VERSION = 1.14.1
```
Comment (wojtek-t) [inline addon-resizer/Makefile:24]: IIUC, 1.14.1 had some bug.
@mm4tt - do you remember?

In k8s we use 1.14.3 - so maybe we could stick to that version here too:
https://github.com/kubernetes/release/blob/master/images/build/cross/Makefile#L23

**Comment by bskiba** [comment]
Please change to 1.14.3 and I'll approve

File: addon-resizer/Makefile (hunk 1 above)
Comment (jkaniuk) [reply addon-resizer/Makefile]: That supposed to be 1.14.4, wrongly amended :-/

Changed to 1.14.3

**Pull request merged**